studio/components/interfaces/Account/AuditLogs.tsx

```diff
@@ -0,0 +1,293 @@
+import * as Tooltip from '@radix-ui/react-tooltip'
+import dayjs from 'dayjs'
+import { useEffect, useState } from 'react'
+
+import { ScaffoldContainerLegacy } from 'components/layouts/Scaffold'
+import Table from 'components/to-be-cleaned/Table'
+import AlertError from 'components/ui/AlertError'
+import { DatePicker } from 'components/ui/DatePicker'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { AuditLog } from 'data/organizations/organization-audit-logs-query'
+import { useOrganizationsQuery } from 'data/organizations/organizations-query'
+import { useProfileAuditLogsQuery } from 'data/profile/profile-audit-logs-query'
+import { useProjectsQuery } from 'data/projects/projects-query'
+import { Alert, Button, IconArrowDown, IconArrowUp, IconRefreshCw } from 'ui'
+import FilterPopover from '../Organization/AuditLogs/FilterPopover'
+import LogDetailsPanel from '../Organization/AuditLogs/LogDetailsPanel'
+
+const AuditLogs = () => {
+  const currentTime = dayjs().utc().set('millisecond', 0)
+  const [dateSortDesc, setDateSortDesc] = useState(true)
+  const [dateRange, setDateRange] = useState({
+    from: currentTime.subtract(1, 'day').toISOString(),
+    to: currentTime.toISOString(),
+  })
+
+  const [selectedLog, setSelectedLog] = useState<AuditLog>()
+  const [filters, setFilters] = useState<{ projects: string[] }>({
+    projects: [], // project_ref[]
+  })
+
+  const { data: projects } = useProjectsQuery()
+  const { data: organizations } = useOrganizationsQuery()
+  const { data, error, isLoading, isSuccess, isError, isRefetching, refetch } =
+    useProfileAuditLogsQuery({
+      iso_timestamp_start: dateRange.from,
+      iso_timestamp_end: dateRange.to,
+    })
+
+  // This feature depends on the subscription tier of the user. Free user can view logs up to 1 day
```

Comment:
If this looks like a good solution, we should also copy it over to the organization level logs screen.

Reply:
yup i think this is good actually 👍